src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+
+    <form>
+      <div class="h1 form-field" data-qa="big">
+        <input
+          data-qa="keypress"
+          type="search"
+          class="input input1"
```

Comment:
що означає клас input1?

```suggestion
          class="searchbar searchbar--small"
```